types/node-resque/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for node-resque 5.4
+// Type definitions for node-resque 5.5.7
```

Comment:
The build says patch version is not allowed. Honestly I'm not sure why, would be interested to know. But if you want build to pass you probably need to change it. 😬

Reply:
Yeah makes sense, patches shouldn't contain API changes (not that semver is set in stone). I'll change those!